compositor_render/src/transformations/web_renderer/browser_client.rs

```diff
@@ -81,6 +53,61 @@ impl BrowserClient {
             frame_data,
             source_transforms,
             resolution,
+            chromium_thread_sender: None,
+        }
+    }
+
+    pub fn set_chromium_thread_sender(&mut self, sender: Sender<ChromiumSenderMessage>) {
+        self.chromium_thread_sender.replace(sender);
+    }
+
+    fn update_source_transforms(&self, msg: &cef::ProcessMessage) {
```

Comment:
You are parsing the message in a function that is named update_sth. I get that trying to maintain single responsibility for everything is often overkill, but this is a textbook case where it should not be mixed.